packages/protocol/contracts/layer1/mainnet/MainnetSignalService.sol

```diff
@@ -1,35 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.24;
-
-import "src/layer1/mainnet/LibFasterReentryLock.sol";
```

Comment:
I think we shall restore MainnetSignalService as it will save gas by using TSTORE